README.md

```diff
@@ -69,15 +72,15 @@ See our [Plugins section](https://github.com/vippsas/vipps-plugins).
 
 ## Status pages
 
-We have several status pages, and we strongly recommend to subscribe to them
-(you can also subscribe to updates for specific incidents):
+We have several status pages and we strongly recommend to subscribe to them.
+You can also subscribe to updates for specific incidents. For example:
 
-* Test environment: https://vipps-test.statuspage.io
-* Production environment: https://vipps.statuspage.io
-* BankID production: https://www.bankid.no/status/
-* BankID preprod: https://bankidpreprod.statuspage.io
+- Test environment: <https://vipps-test.statuspage.io>
+- Production environment: <https://vipps.statuspage.io>
+- BankID production: <https://www.bankid.no/status/>
+- BankID preprod: <https://bankidpreprod.statuspage.io>
```

Comment:
Should all links be in angle brackets? 🧐

Reply:
The markdown editor checker complained about them, so I was trying to get them out of the way so I'd be sure to see the more important errors. But we can also ignore that error check.